src/lib/hooks/useResizeObserver.ts

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useRef } from 'react';
+import ResizeObserver from 'resize-observer-polyfill';
+
+import { useLatest } from '../../lib';
+
+export function useResizeObserver(element: HTMLElement | null, callback: ResizeObserverCallback) {
```

Comment:
We can use `useResizeObserver` from `react-hookz` instead:
https://react-hookz.github.io/web/?path=/docs/sensor-useresizeobserver--example